WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -601,7 +601,7 @@ private void LogCoinJoinSummary(ImmutableArray<AliceClient> registeredAliceClien
 
 		var filteredCoins = coins
 			.Where(x => parameters.AllowedInputAmounts.Contains(x.Amount))
-			.Where(x => parameters.AllowedInputTypes.Any(t => x.ScriptPubKey.IsScriptType(t)))
+			.Where(x => parameters.AllowedInputScriptTypes.Any(t => x.ScriptPubKey.IsScriptType(t)))
```

Comment:
This is okay.

What is missing in the CoinJoinClient side it to not participate in coinjoins where p2wpkh are not allowed (because wasabi cannot handle that and will disrupt the p2tr rounds)

Reply:
Implemented in https://github.com/zkSNACKs/WalletWasabi/pull/8831/commits/9f43f7085691e5ace45fbd5f7ff2d3042380fdd0.

Reply:
It looks like this particular change made the CodeScene check fail. :(

Reply:
Don't take it seriously, it has never written a single line of code.